test/flutter/templates.test.ts

```diff
@@ -39,6 +39,9 @@ describe('Flutter code templates', () => {
         "await SentryFlutter.init(
             (options) {
               options.dsn = 'my-dsn';
+              // Adds request headers and IP for users, for more info visit:
+              // https://docs.sentry.io/platforms/dart/guides/flutter/data-management/data-collected/ for more info
```

Comment:
`l` A bit redundant having fore more info after already stating more info before the link. Applies to all changes in this file.
```suggestion
              // https://docs.sentry.io/platforms/dart/guides/flutter/data-management/data-collected/
```